components/Icons/StyledSvg.js

```diff
@@ -64,3 +64,18 @@
   <title>step-backward</title>
   <path d="M19,5V19H16V5M14,5V19L3,12" />
 </svg>;
+
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <title>microphone-outline</title>
+  <path d="M17.3,11C17.3,14 14.76,16.1 12,16.1C9.24,16.1 6.7,14 6.7,11H5C5,14.41 7.72,17.23 11,17.72V21H13V17.72C16.28,17.23 19,14.41 19,11M10.8,4.9C10.8,4.24 11.34,3.7 12,3.7C12.66,3.7 13.2,4.24 13.2,4.9L13.19,11.1C13.19,11.76 12.66,12.3 12,12.3C11.34,12.3 10.8,11.76 10.8,11.1M12,14A3,3 0 0,0 15,11V5A3,3 0 0,0 12,2A3,3 0 0,0 9,5V11A3,3 0 0,0 12,14Z" />
+</svg>;
+
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <title>microphone</title>
+  <path d="M12,2A3,3 0 0,1 15,5V11A3,3 0 0,1 12,14A3,3 0 0,1 9,11V5A3,3 0 0,1 12,2M19,11C19,14.53 16.39,17.44 13,17.93V21H11V17.93C7.61,17.44 5,14.53 5,11H7A5,5 0 0,0 12,16A5,5 0 0,0 17,11H19Z" />
+</svg>;
+
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <title>volume-high</title>
+  <path d="M14,3.23V5.29C16.89,6.15 19,8.83 19,12C19,15.17 16.89,17.84 14,18.7V20.77C18,19.86 21,16.28 21,12C21,7.72 18,4.14 14,3.23M16.5,12C16.5,10.23 15.5,8.71 14,7.97V16C15.5,15.29 16.5,13.76 16.5,12M3,9V15H7L12,20V4L7,9H3Z" />
+</svg>;
```

Comment:
What does this whole document do?

Reply:
Yust storing my pathes ;-) 
I'll delete it later.